.github/labeler.yml

```diff
@@ -8,6 +8,9 @@ CI:
 - any: ['.appveyor.yml']
 - any: ['.codecov.yml']
 
+documentation:
+- any: ['*/doc/*']
```

Comment:
I think this should be `all:`, otherwise any PR that updates documentation will have this label, which doesn't look right.

Reply:
Done: https://github.com/vim/vim/pull/13507